rag/ddl_indexer.py

```
@@ -0,0 +1,31 @@
+"""Utility to index DDL statements into a Chroma collection."""
+
+from pathlib import Path
+from langchain_text_splitters import RecursiveCharacterTextSplitter
```

Pylint (3.3.7)

[error] Unable to import 'langchain_text_splitters'

(E0401)